lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerController.pm

```diff
@@ -127,7 +149,9 @@ sub create_new_partition_table {
     my ($self, $args) = @_;
     $self->get_expert_partitioner_page()->select_disk($args->{name});
     $self->get_expert_partitioner_page()->select_create_partition_table();
-    $self->get_deleting_current_devices_warning()->press_yes();
+    if ($self->get_deleting_current_devices_warning()->is_shown()) {
```

Comment:
what happens if it is shown when is not supposed to be shown? I think we should be deterministic using some kind of variable instead of checking the screen (like what we were doing with needling). Then we would have all the control on the test to avoid false negatives.

Reply:
Makes sense. It is always better to have solid expectations. Changed.